spec/firebase_app_distribution_api_client_spec.rb

```diff
@@ -205,7 +205,7 @@
       api_client.enable_access("app_id", "release_id", ["testers"], ["groups"])
     end
 
-    it 'posts when groupIds are defined and tester emails is nil' do
+    it 'posts when group_ids are defined and tester emails is nil' do
```

Comment:
Is rubocop making you change this? If not, you can leave it as `groupIds` since that's how it's passed in the payload.